### ISSUE TITLE
Checksum error when upgrading through Homebrew

### ISSUE THREAD

**Issue by aaparmeggiani** [glushchenko/fsnotes#843]
**Describe the bug**
Checksum error when trying to upgrade using Homebrew.

**To Reproduce**
```
% brew update
Already up-to-date.
%brew cask upgrade fsnotes
Updating Homebrew...
==> Auto-updated Homebrew!
==> Upgrading 1 outdated package:
fsnotes 3.8.2 -> 4.1.3
==> Upgrading fsnotes
==> Downloading https://github.com/glushchenko/fsnotes/releases/download/4.1.3/FSNotes_4.1.3.zip
Already downloaded: /Users/xxx/Library/Caches/Homebrew/downloads/3018171411972c08b9323f997d16c11e472a4001c9feb6797247854353986b27--FSNotes_4.1.3.zip
==> Verifying SHA-256 checksum for Cask 'fsnotes'.
==> Note: Running `brew update` may fix SHA-256 checksum errors.
==> Purging files for version 4.1.3 of Cask fsnotes
Error: Checksum for Cask 'fsnotes' does not match.
Expected: 7d96b80c539ac1d291161f40344dbe192be790538cb02bf0a738303705bb441f
  Actual: c17f9a93af86404b991ea1439d26b4550641292d2c600e1071c728af5ae467d7
    File: /Users/xxx/Library/Caches/Homebrew/downloads/3018171411972c08b9323f997d16c11e472a4001c9feb6797247854353986b27--FSNotes_4.1.3.zip
```
**Expected behavior**
Be able to upgrade without errors

**Desktop:**
 - OS: macOS 
 - OS version: 10.15.3
 - FSNotes version: 3.8.2

**Comment by coder202** [comment]
Same checksum error.
FSNotes 4.1.2 -> 4.1.3

**Comment by glushchenko** [comment]
Fixed in 4.1.5
